widget/ui/src/components/StatusDrawer/Item.tsx

```diff
@@ -0,0 +1,35 @@
+import type { Data } from './StatusDrawer.types';
+
+import React from 'react';
+
+import { ChevronDownIcon, ChevronUpIcon } from '../../icons';
+import { Collapse } from '../Collapse';
+import { Divider } from '../Divider';
+import { StatusCard } from '../StatusCard';
+
+import { CollapseContainer } from './StatusDrawer.styles';
+
+export function Item({
+  description,
+  status,
+  title,
+  content,
+  onClick,
+  expanded,
```

Comment:
I think destructuring (too many properties) the props in the function body makes it easier to read